src/pages/challenges/component.js

```diff
@@ -0,0 +1,81 @@
+import React, {useEffect, useState} from "react";
+import Header from "../../components/header";
+import Footer from "../../components/footer";
+import Search from "../../components/search";
+import ChallengeDetails from "../../components/challengeDetails";
+import {Tabs} from "antd";
+import {Link} from "react-router-dom";
+import CNST from "../../constants";
+import settingsIcon from "../../images/settings.svg";
+import userImg from "../../images/user.png";
+import styles from "./_.module.css";
+
+const {TabPane} = Tabs;
+
+const SettingsIcon = ({className}) => (
+  <Link to={CNST.ROUTES.SETTINGS}>
+    <img src={settingsIcon} alt="Settings" className={className} />
+  </Link>
+);
+
+const UserImage = ({userImg, className}) => (
+  <img src={userImg} alt="User" className={className} />
+);
+
+export const Challenges = ({fetchChallenges, challenges, fetching}) => {
+  const [tab, setTab] = useState();
+  const [search, setSearch] = useState("");
+
+  useEffect(() => {
+    if (!fetching && !challenges) {
```

Comment:
why do you need to have this checking?
If you receive challenges on the main screen it seems you are not going get challenges on this page, you just reuse challenges on all screens